src/main/java/it/gov/pagopa/pu/sil/enums/legacy/IngestionFlowFileLegacyStatus.java

```diff
@@ -0,0 +1,31 @@
+package it.gov.pagopa.pu.sil.enums.legacy;
+
+import java.util.Arrays;
+
+public enum IngestionFlowFileLegacyStatus {
+  UPLOADED("UPLOADED", "FILE_SCARICATO"),
+  PROCESSING("PROCESSING", "FILE_IN_CARICAMENTO"),
+  COMPLETED("COMPLETED", "FILE_CARICATO"),
+  ERROR("ERROR", "ERROR_LOAD"),
+  WAITING_FILE("WAITING_FILE", "WAITING_FILE") /** no such value exists in the legacy system */
+  ;
+
+  private String value;
```

Comment:
let's simplify using our enum
```suggestion
  UPLOADED(IngestionFlowFileStatusEnum.UPLOADED, "FILE_SCARICATO"),
  PROCESSING(IngestionFlowFileStatusEnum.PROCESSING, "FILE_IN_CARICAMENTO"),
  COMPLETED(IngestionFlowFileStatusEnum.COMPLETED, "FILE_CARICATO"),
  ERROR(IngestionFlowFileStatusEnum.ERROR, "ERROR_LOAD"),
  WAITING_FILE(IngestionFlowFileStatusEnum.WAITING_FILE, "WAITING_FILE") /** no such value exists in the legacy system */
  ;

  private IngestionFlowFileStatusEnum value;
```